### ISSUE TITLE
segfault:  rexgen "\0"

### ISSUE THREAD

**Issue by frank-dittrich** [teeshop/rexgen#11]
```
$ rexgen "\0"
Segmentation fault (core dumped)
$ rexgen "x\0"
Segmentation fault (core dumped)
$ rexgen "\0x"
Segmentation fault (core dumped)
```

**Comment by frank-dittrich** [comment]
I still see the same problem, see my comment to your commit:
https://github.com/teeshop/rexgen/commit/18fff68dd7ab1a72af0eddebd95eeba78148f1cd#commitcomment-11742022

**Comment by teeshop** [comment]
damn... I forgot to push the release :-(

**Comment by teeshop** [comment]
fixed in 1.2.5
